src/integrationTest/resources/application-integration.yaml

```diff
@@ -34,6 +34,8 @@ idam:
     username: ${TEST_WA_LAW_FIRM_USERNAME:some_user@hmcts.net}
     password: ${TEST_WA_LAW_FIRM_PASSWORD:password}
 
+environment: ${ENVIRONMENT:arm}
```

Comment:
Should this be `arm` by default?  Surely this would fail for anyone where they don't need minikube tunnel?  Think the default could be just `local` and then an m1/2 Arm arch could be named `local-arm-arch` so it is more descriptive.

Reply:
Then we need to update the Readme that this is what you'd have to supply as an env var if you're using an Arm arch.